.github/workflows/build-push.yml

```diff
@@ -8,44 +8,59 @@ on:
     tags:
       - "v*.*.*"
     paths-ignore:
-      - 'README.md'
+      - "README.md"
   pull_request:
 
-# Defines two custom environment variables for the workflow. These are used for the Container registry domain, and a name for the Docker image that this workflow builds.
+# Defines two custom environment variables for the workflow.
+# These are used for the Container registry domain, and a name
+# for the Docker image that this workflow builds.
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: substratusai/lingo
 
-# There is a single job in this workflow. It's configured to run on the latest available version of Ubuntu.
+# There is a single job in this workflow.
+# It's configured to run on the latest available version of Ubuntu.
 jobs:
-  build-and-push-image:
+  container:
```

Comment:
container as job name seems less descriptive than `build-and-push-image`?

Reply:
See other response